packages/cli/test/unit/commands/link.test.ts

```diff
@@ -306,5 +306,28 @@ describe('link', () => {
         process.chdir(originalCwd);
       }
     });
+    it('should respect --yes', async () => {
+      const cwd = fixture('single-remote');
+      try {
+        process.chdir(cwd);
+        await fs.rename(join(cwd, 'git'), join(cwd, '.git'));
+
+        useUser();
+        useProject({
+          ...defaultProject,
+          name: 'single-remote',
+          id: 'single-remote',
+        });
+        useTeams('team_dummy');
+        client.setArgv('--yes');
+        const linkPromise = link(client);
+        expect(client.stderr).not.toOutput('Do you want to connect "origin"');
+        await expect(client.stderr).toOutput('Linked to');
+        await expect(linkPromise).resolves.toEqual(0);
+      } finally {
+        await fs.rename(join(cwd, '.git'), join(cwd, 'git'));
+        process.chdir(originalCwd);
+      }
+    });
```

Comment:
**suggestion (soft-blocking):** add test for multiple remotes

In this or a follow-up PR: Can we add a test for multiple remotes that prefer `origin` when `origin` is not the first item in the list?

Reply:
Added test!